binary/script/build_and_pr.sh

```diff
@@ -42,8 +41,11 @@ function check_github_token() {
 # check for changes to iPXE files
 function changes_detected() {
     local file="${1:-sha512sum.txt}"
+    local test="${file}.test" # .gitignore contains "*.test"
```

Comment:
Should we add an entry to .gitignore for this file? not that it really matters as it should always get overwritten

Reply:
Another option is to make `create_checksums`write to stdout, then you could do.

```
if create_checksums | diff "$file" -; then
```

Reply:
It already matches an existing .gitignore pattern.

Reply:
oh, should've looked. great!

Reply:
I did note it in the comment. Was that not clear?

Reply:
Removed anyway, resolving this thread.